YOLO.xcodeproj/project.pbxproj

```diff
@@ -87,11 +77,6 @@
 		63A946D8271800E20001C3ED /* Models */ = {
 			isa = PBXGroup;
 			children = (
-				6381D2132B7817C200ABA4E8 /* yolov8l.mlpackage */,
```

Comment:
Again, we don't want to eliminate these references!

Reply:
I changed those lines back and restored the model references.